yt_dlp/extractor/aenetworks.py

```diff
@@ -227,6 +227,7 @@ class AENetworksCollectionIE(AENetworksListBaseIE):
             'title': 'America The Story of Us',
         },
         'playlist_mincount': 12,
+        'skip': 'probably geo-restricted, redirects to aenetworks.tv for me',
```

Comment:
Don't add `skip` for geo-retriction

Reply:
In other cases skip was present for geo-restriction, that was the reason I added it in cases of geo restriction. Should I remove the skip tag for all cases of geo-restriction?
That being said: in this case, I honestly don't know if it's geo-restricted - there is no message about it being geo-restricted, but sometimes geo-restrictions aren't too obvious. Someone in the US would have to test that link.